src/core/src/bootstrap/CoreUtility.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2020 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Requires Python 2.7+
+import re
+
+
+class CoreUtility(object):
```

Comment:
The use of a utility class is a 'code smell'. It violates the Single Responsibility Principle. We'd moved away from Utility in core for this specific reason and it's good to not reintroduce it as debt. (Handler continues to have it as debt in a Utility class).

@rane-rajasi can provide more historical context from older conversations I've had with her on this.

Also, as an educational thing, it's good to look up SOLID principles for general code-writing goodness: https://www.freecodecamp.org/news/solid-principles-explained-in-plain-english/

Reply:
This may be 'VersionComparer' as a class under CoreLogic. This isn't bootstrap functionality or core to any bootstrap code path (which should be implicitly light weight.

Reply:
create a new class - versioncomparator instead of it being a static class